components/document-list/template.njk

```diff
@@ -12,11 +12,18 @@
         {{ item.data.description | markdown("inline") | safe }}
       </p>
       {% endif %}
-      {% if item.data.date %}
+      {% if item.data.date or item.data.tags %}
       <ul class="app-document-list__item-metadata">
+        {% if item.data.date %}
         <li class="app-document-list__attribute">
           <time datetime="{{ item.data.date | date }}">{{ item.data.date | date("d LLLL y") }}</time>
         </li>
+        {% endif %}
+        {% if item.data.tags %}
+        <li class="app-document-list__attribute">
+          Tags: {{ item.data.tags | join(", ") }}
+        </li>
+        {% endif %}
```

Comment:
I hadn't thought of including these on the listing pages. Could be helpful though! Should we make it configurable somehow (eg a `showTags` param)?  I can see an argument for hiding the list of tags on the tag page itself, as most of them will all be the same tag...

Reply:
Maybe if it’s requested, rather than adding too many options?